samples/AvatarWithRoundedCorner/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿
+
+namespace AvatarWithRoundedCorner
+{
+    using System;
+    using System.Numerics;
+    using ImageSharp;
+    using SixLabors.Shapes;
+
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
Isn't it better to have a single samples project, with different code examples residing in different classes? (Launching them with some kind of sample switcher in `Main()`)